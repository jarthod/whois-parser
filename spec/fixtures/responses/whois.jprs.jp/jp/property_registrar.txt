[ JPRS database provides information on network administration. Its use is    ]
[ restricted to network administration purposes. For further information,     ]
[ use 'whois -h whois.jprs.jp help'. To suppress Japanese output, add'/e'     ]
[ at the end of command, e.g. 'whois -h whois.jprs.jp xxx/e'.                 ]
Domain Information: [ドメイン情報]
[Domain Name]                   MAILMATE.JP

[登録者名]                      ネットオウル株式会社
[Registrant]                    Netowl,Inc.

[Name Server]                   nitin.ns.cloudflare.com
[Name Server]                   rosa.ns.cloudflare.com
[Signing Key]                   

[登録年月日]                    2019/06/17
[有効期限]                      2025/06/30
[状態]                          Active
[最終更新]                      2024/07/01 01:05:03 (JST)

Contact Information: [公開連絡窓口]
[名前]                          ネットオウル株式会社
[Name]                          Netowl, Inc.
[Email]                         whois@netowl.jp
[Web Page]                       
[郵便番号]                       
[住所]                           
[Postal Address]                 
[電話番号]                      075-256-8553
[FAX番号]                       