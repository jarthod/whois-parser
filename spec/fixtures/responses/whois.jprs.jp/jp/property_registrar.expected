#registrar
  %s %CLASS{registrar}
  %s.id == nil
  %s.name == "Netowl,Inc."
  %s.organization == "Netowl, Inc."
  %s.url == nil 